rubygem-foreman_openscap/rubygem-foreman_openscap.spec

```diff
@@ -83,11 +83,14 @@ mkdir -p %{buildroot}%{foreman_bundlerd_dir}
 %{gem_instdir}/db
 %{gem_instdir}/config
 %{gem_instdir}/locale
+%foreman_apipie_cache_plugin
+%foreman_apipie_cache_foreman
```

Comment:
Isn't this only needed for foreman itself and not plugins?

Reply:
Both are needed, first packages the actual files while the other adds a [link from core](http://koji.katello.org/kojifiles/work/tasks/3491/43491/build.log) to these files.

Reply:
Looks like you're right and other plugins do this too.